### ISSUE TITLE
no-single-variables-to-translate false positive

### ISSUE THREAD

**Issue by Jack-Works** [lingui/eslint-plugin#104]
For code:

```tsx
<Trans>
    <span style={{ color: ... }}>{websiteCount}</span>{' '}
    <Plural value={websiteCount} one="Website" other="Websites" />
</Trans>
```

It will report an error, but this is valid:

```
msgid "<0>{websiteCount}</0> {websiteCount, plural, one {Website} other {Websites}}"
```
